client/src/translations/en.json

```diff
@@ -222,7 +222,7 @@
   "min_value": "Min: {val}",
   "min": "Min",
   "misc": "Other options",
-  "multiple": "Multiple",
+  "multiple": "Multiple values",
```

Comment:
I suggest we change the key to "multiple_values".

Reply:
Was meant as a suggestion, I could still merge the fix if we want it this Friday afternoon!